typescript/cli/src/deploy/warp.ts

```diff
@@ -137,6 +140,7 @@ export async function runWarpRouteDeploy({
   };
 
   await runDeployPlanStep(deploymentParams);
+  await validateAndConfirmOwners(multiProvider, warpDeployConfig);
```

Comment:
i think this does not cover cosmos chain right? should we also add support for that later?